releng/server-runtime/src/main/resources/dirigible.properties

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2017 SAP and others.
```

Comment:
Need to change the license header

Reply:
It is identical to `server` option.
I suppose it should be changed there as well then.

Reply:
Yes! You can take from `sap-kyma-runtime`.